addon/session-stores/adaptive.js

```diff
@@ -133,7 +133,9 @@ export default Base.extend({
       const options = this.getProperties('cookieDomain', 'cookieName', 'cookieExpirationTime', 'cookiePath');
       options._fastboot = this.get('_fastboot');
       options._cookies = this.get('_cookies');
+
       store = this._createStore(Cookie, options);
+      this.set('cookieExpirationTime', store.get('cookieExpirationTime'));
```

Comment:
Made a small change. If `cookieExpirationTime` is not set here, then it stays `null` because the setter in `proxyToInternalStore` is never called.